servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -304,6 +309,16 @@ public static BackOffPolicy ofInstant() {
             return new BackOffPolicy(null, ZERO, null, null, false, 3);
         }
 
+        /**
+         * Creates a new {@link BackOffPolicy} that retries failures instantly up-to provided max retries.
+         * 
+         * @param maxRetries the number of retry attempts for this {@link BackOffPolicy}.
+         * @return a new {@link BackOffPolicy} that retries failures instantly up-to provided max retries.
+         */
+        public static BackOffPolicy ofInstant(final int maxRetries) {
```

Comment:
perhaps `immediateRetry`? When I first read the description I interpreted as unconditionally fail rather than retry immediately upon failure.

Reply:
how about `ofImmediate` to much the API lingo used in the rest of `BackOffPolicy` class?

Reply:
Yes, that seems reasonable.